Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -3352,7 +3353,7 @@ def main():  # pragma: no cover
 
     connection_args = get_connection_args(params)
 
-    base_url = f"https://{params['host'].replace('https://', '')}:{params['port']}/"
+    base_url = f"https://{connection_args['host']}:{connection_args['port']}/"
```

Comment:
As the `base_url` is used only in `splunk-notable-event-edit` please move it to the `elif command == 'splunk-notable-event-edit':` section